app/src/components/googleMap.tsx

```diff
@@ -130,6 +133,75 @@ class SimpleMap extends React.Component<MapProp, MapState> {
         this.googleMaps = maps;
     }
     
+    fullScreenClick() {
+        let mainFooter = document.getElementById('mainfooter');
+        let nav = document.getElementById('main-nav');
+        let navItem = document.getElementById('nav-items');
+        let widgetContainer = document.getElementById('wrapperContainer');
+        let mainContainer = document.getElementById('main-container');
+        let text = document.getElementsByClassName('filter-title');
+        
+        if (text.length !== 0) {
+            for (let i = 0; i < text.length; i++) {
+                
+                if ((text.item(i) as HTMLElement).style.color !== 'rgb(0, 0, 0)') {
+                    (text.item(i) as HTMLElement).style.color = 'rgb(0, 0, 0)';
+                } else {
+                    (text.item(i) as HTMLElement).style.color = '#dfdfdf';
+                }
+            }
+        }
+        if (!this.state.isFullScreen) {
+            this.setState({
+                isFullScreen: true
+            });
+        } else {
+            this.setState({
+                isFullScreen: false
+            });
+        }
```

Comment:
what??? update to:

this.setState({
                isFullScreen: !this.state.isFullScreen
            });